packages/web/config/ibc-assets.ts

```diff
@@ -163,6 +163,44 @@ export const IBCAssetInfos: (IBCAsset & {
     coinMinimalDenom: "uscrt",
     isVerified: true,
   },
+  {
+    counterpartyChainId: "secret-4",
+    sourceChannelId: "channel-476",
+    destChannelId: "channel-39",
+    coinMinimalDenom: "cw20:secret1qfql357amn448duf5gvp9gr48sxx9tsnhupu3d:SHD",
+    ics20ContractAddress: "secret1tqmms5awftpuhalcv5h5mg76fa0tkdz4jv9ex4",
+  },
+  {
+    counterpartyChainId: "secret-4",
+    sourceChannelId: "channel-476",
+    destChannelId: "channel-39",
```

Comment:
This looks incorrect to me. I'm getting a different counterparty(secretnetwork) channel ID...
<img width="257" alt="image" src="https://user-images.githubusercontent.com/95667791/201205044-c796b228-3412-4c60-8599-3b4cf22217be.png">

Reply:
looks to be channel-44 in the other file. So ya probably just a repeated typo in this file.

Reply:
Yes looks like another typo @assafmo can you just change the IDs to 44 as is in our document?

Reply:
Fixed